home/management/commands/importgradedata.py

```diff
@@ -43,8 +43,11 @@ def handle(self, *args, **options):
             for row in reader:
                 self.add_grade(row)
 
-        grades = Grade.unfiltered.bulk_create(self.grades)
-        self.stdout.write(f"Done, added {len(grades)} grades")
+        try:
+            grades = Grade.unfiltered.bulk_create(self.grades)
+            self.stdout.write(f"Done, added {len(grades)} grades")
+        except Exception as e:
+            print(e)
```

Comment:
?

Reply:
lol that was me messing around with stuff. guess I forgot to change branches